src/relay/backend/contrib/cmsisnn/relay_to_tir.cc

```diff
@@ -196,12 +227,13 @@ class RelayToTIRVisitor : public MixedModeMutator {
 
     tvm::Array<PrimExpr> call_ext_args = {tir::StringImm(cmsisnn_api), input, filter, multiplier};
     if (bias_add_call) {
+      tir::Var bias = buffer_creator_.GetBufferVar("bias");
       call_ext_args.push_back(bias);
     }
     call_ext_args.push_back(shift);
     call_ext_args.push_back(output);
 
-    std::string context_buffer_name = "NULL";
+    PrimExpr buffer_var = tir::StringImm("NULL");
```

Comment:
nit: Like other places, it can be renamed to context_buffer_var for better clarity.

Reply:
Done